src/app.ts

```diff
@@ -0,0 +1,90 @@
+import { Probot, Context } from 'probot';
+
+interface PullRequestInfo {
+  owner: string
+  repo: string
+  pull_number: number
+}
+
+const getPRInfo = async (context: Context, {
+  owner, repo, pull_number
+}: PullRequestInfo) => {
+  const {
+      data: {
+        commits,
+        head
+      }
+    } = await context.octokit.pulls.get({
+      owner, repo, pull_number
+    });
+  
+  return { prNumCommits: commits, prHeadSha: head.sha, prBranchName: head.ref }
+}
+
+const createEmptyCommit = async (
+  context: Context,
+  {
+    owner,
+    repo,
+    prHeadSha
+  }: PullRequestInfo & { prHeadSha: string }
+) => {
+  const { data: { sha } } = await context.octokit.git.createCommit({
+        owner,
+        repo,
+        message: 'empty commit',
+        tree: '', //TODO To check how to get the SHA of the tree object this commit points to
```

Comment:
🏗️ To check how to get the SHA of the tree object this commit points to

Reply:
Will we need all the logic in this discussion necessary for getting the tree? https://github.com/octokit/rest.js/discussions/1938#discussioncomment-132896

Reply:
Inspired by this issue comment: https://github.com/renovatebot/renovate/issues/4408#issuecomment-664012687